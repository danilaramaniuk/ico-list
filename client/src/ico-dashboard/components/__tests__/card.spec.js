import React from 'react';
import { renderWithTheme, getMockedContributions } from '~/__tests__/helpers';
import Card from '../card';

const props = {
  item: getMockedContributions()[0],
};

describe('Card component', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<Card {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with index correctly', () => {
    const propsWithIndex = { ...props, index: 1 };
    const tree = renderWithTheme(<Card {...propsWithIndex} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
