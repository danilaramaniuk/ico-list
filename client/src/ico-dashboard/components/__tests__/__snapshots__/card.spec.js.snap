// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders correctly 1`] = `
<div
  className="sc-bdVaJa cxEflh"
>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      currency
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      LTC
    </div>
  </div>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      value
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      1822819
    </div>
  </div>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      address
    </div>
    <div
      className="sc-htpNat iNMcUK"
    >
      txid
    </div>
  </div>
  <div
    className="sc-ifAKCX gRAkOE"
  >
    <div
      className="sc-bxivhb kwdKHT"
    >
      rnu2xb48synW6Y1AU8ZT85OX0CWBD21N
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      4911922634ee442f6704ed6cf5f860dbe70e78d1c20414d6d6f2eb4bc81e1641
    </div>
  </div>
</div>
`;

exports[`Card component renders with index correctly 1`] = `
<div
  className="sc-bdVaJa cxEflh"
>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      index
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      2
    </div>
  </div>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      currency
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      LTC
    </div>
  </div>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      value
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      1822819
    </div>
  </div>
  <div
    className="sc-bwzfXH eHRqiU"
  >
    <div
      className="sc-htpNat iNMcUK"
    >
      address
    </div>
    <div
      className="sc-htpNat iNMcUK"
    >
      txid
    </div>
  </div>
  <div
    className="sc-ifAKCX gRAkOE"
  >
    <div
      className="sc-bxivhb kwdKHT"
    >
      rnu2xb48synW6Y1AU8ZT85OX0CWBD21N
    </div>
    <div
      className="sc-bxivhb kwdKHT"
    >
      4911922634ee442f6704ed6cf5f860dbe70e78d1c20414d6d6f2eb4bc81e1641
    </div>
  </div>
</div>
`;
